backend/Squirrel.Core/Squirrel.Core.WebAPI/Controllers/AuthController.cs

```diff
@@ -2,11 +2,12 @@
 using Microsoft.AspNetCore.Mvc;
 using Squirrel.Core.BLL.Interfaces;
 using Squirrel.Core.Common.DTO.Auth;
+using Squirrel.Core.Common.DTO.Users;
 
 namespace Squirrel.Core.WebAPI.Controllers;
 
 [ApiController]
-[Route("api/[controller]")]
+[Route("[controller]")]
```

Comment:
are you sure that we don't need 'api' prefix?

Reply:
in `SampleController` we don't have, so to be consistent

Reply:
Leave it, please, we will have troubles with deploy without it)

Reply:
returned